docs/community/tutorials-contest/2022/erc20-as-subnet-gas-token/README.md

```diff
@@ -0,0 +1,468 @@
+
+#### Test Relayer
+
+In this video on the left terminal I am using our custom scripts to interact with chains and on the right terminal I am using our relay to create the cross chain communication.
+
+![Test relayer with scripts](./assets/2.gif)
+
+##### What Happens on the Video?
+
+- Check balances on both chains
+- Start the relayer
+- An already processed event appears on the console of the relayer therefore it does not get processed
+- Check the balances to make sure already processed event is not processed
+- Lock 20 ERC20s token from avax and see the updated balances of user and the bridge on avax
+- Relayer observes the transaction and sends a transaction to mint native tokens on Subnet
+- Check the balances on both Subnet and avax. As expected our Subnet native token balance increases by 20
+- Burn 5 native tokens from Subnet and see the updated balance of the user
+- Relayer observers the transaction and sends a transaction to release ERC20 tokens on avax
+- Check balance on both Subnet and Avax. As expected our ERC20 balance increases by 5 and bridge's decreases by 5
+
+#### Test Relayer for Old Events
+
+![Test relayer for old events](./assets/1.gif)
+
+##### What Happens on the Video?
+
+- Start by a lock transaction from avax with amount 40. As stated in the video this transaction was sent when the relayer was not working. Therefore, it is not processed and it will not be processed when we start the relayer with `node relayer.js` because there has been many blocks after it
+- Check balances on both chains
+- Start the relayer with `node relayer.js` to show that the event is not getting processed. Printed events are events that happened on the Subnet. Since there is no blocks building on my local Subnet other than my own, my old burns are considered old and therefore shown
+- Start the relayer with `node relayer.js <blockNumber>` to show that event will be processed and will be printed as "OLD: "
+- Check balances on both chains to confirm that old lock event on avax has been processed by relayer and tokens have been minted on avax
+
+## Troubleshoot Common Issues
+
+Things to check out;
+
+- Error while compiling contracts
+  - You have run `npm i @openzeppelin/contracts`.
+- Error while running scripts
+  - Both accounts on both chains have some native token so that they can send transactions.
+  - Folder structures and file names are as suggested. In our scripts we access the contract abis and bytecodes directly from the files that are created by hardhat. Those files are created according to your file structure and if you changed the structure, imports might fail.
+  - You have your private keys inside the .env file and you have downloaded dotenv package by running `npm i dotenv`.
+  - Your Subnet has NativeMinter precompile with bridgeAdmin account as the admin.
+  - You have created a contractAddresses.js file inside the variables folder. If you did not create this file, deploy.js would fail.
+
+## Security and Maintenance
```

Comment:
👍

Reply:
I guess this is an approval for `Security and Maintenance` content 🥳 🥳

Reply:
Yup. Looks good to me. Readers will appreciate any reference links for further reading 😉